hammer/drc/icv/defaults.yml

```diff
@@ -1,22 +1,22 @@
 # Default settings for ICV DRC, for project/technology configuration and overriding.
 drc.icv:
     # Location of the binary
-    icv_drc_bin: "${synopsys.synopsys_home}/icv/${drc.icv.version}/bin/LINUX.64/icv"
+    icv_drc_bin: "${synopsys.synopsys_home}/icvalidator/${drc.icv.version}/bin/LINUX.64/icv"
     icv_drc_bin_meta: lazysubst
 
-    icv_vue_bin: "${synopsys.synopsys_home}/icv/${drc.icv.version}/bin/LINUX.64/icv_vue"
+    icv_vue_bin: "${synopsys.synopsys_home}/icvalidator/${drc.icv.version}/bin/LINUX.64/icv_vue"
     icv_vue_bin_meta: lazysubst
 
     icvwb_bin: "${synopsys.synopsys_home}/icvwb/${drc.icv.icvwb_version}/bin/icvwb"
     icvwb_bin_meta: lazysubst
 
-    ICV_HOME_DIR: "${synopsys.synopsys_home}/icv/${drc.icv.version}"
```

Comment:
@elamdf Requested a comment to users that hints their directory may be /icv/ instead of /icvalidator/